### DISCUSSION TITLE
Issues creating config file from Keras

### DISCUSSION THREAD

**Discussion by KevinMonahanUWM** [fastmachinelearning/hls4ml#1046] [Q&A]
I am trying to convert a model that was created in Matlab into hsl code. I have converted the matlab neural network to tensorflow and have followed the steps generated by the matlab code to save this models as a .keras file. I then load this file in and attempt to create a config based off the model. However, I keep running into errors with with the keys and specific ways this tool converts from keras to hls. Is there anything that I can do fix these issues. I am running all of this code in a VM running linux.

**Comment by vloncar** [comment]
Please use an [issue template](https://github.com/fastmachinelearning/hls4ml/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=) when reporting problems. It's easier to provide assistance if we know what was done
